packages/salesforcedx-vscode-core/src/conflict/timestampConflictDetector.ts

```diff
@@ -28,16 +28,20 @@ export class TimestampConflictDetector {
     this.diffs = Object.assign({}, TimestampConflictDetector.EMPTY_DIFFS);
   }
 
-  public createDiffs(
+  public getDiffs(): DirectoryDiffResults {
+    return this.diffs;
+  }
+
+  public async createDiffs(
+    username: string,
```

Comment:
why did we need to add in the username parameter here?

Reply:
This is just so we can match a signature exported by MetadataCacheExecutor. It uses the type MetadataCacheCallback which requires a username parameter. So you're correct that the username parameter is unused, but I just added it in instead of changing the callback type (which I think would also be doable).